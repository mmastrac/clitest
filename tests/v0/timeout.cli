#!/usr/bin/env clitest --v0

$ echo "Goodnight, forever!" && sleep 60
%TIMEOUT 100ms
%EXIT timeout
! Goodnight, forever!
