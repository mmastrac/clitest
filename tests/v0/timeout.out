Running <root>/timeout.cli ...

echo "Goodnight, forever!" && sleep 60
---
Goodnight, forever!
Process took too long!
---
âœ… OK (timed out)

<root>/timeout.cli PASSED
